src/components/search/TopBar.tsx

```diff
@@ -12,61 +12,70 @@ import LottieView from 'lottie-react-native'
 
 import IconCaretRight from '../../assets/images/iconCaretRight.svg'
 import IconRemove from '../../assets/images/iconRemove.svg'
-import { useColor, useSpecialColor, useTheme } from '../../utils/theme'
+import { useColor, useTheme } from '../../utils/theme'
 import { useDispatchWebAction } from '../../hooks/useWebAction'
 import { MessageType } from '../../message'
-
+import {
+  getTagSearchRoute
+} from '../../utils/routes'
 import { getSearchQuery, getSearchResultQuery } from '../../store/search/selectors'
 import { updateQuery } from '../../store/search/actions'
 import useSearchHistory from '../../store/search/hooks'
+import { usePushSearchRoute } from './utils'
 
 
 const IS_IOS = Platform.OS === 'ios'
 
 const styles = StyleSheet.create({
   topBar: {
-    height: IS_IOS ? 85 : 55,
+    height: IS_IOS ? 86 : 55,
```

Comment:
86 or 87?

Reply:
This has a 1 px border bottom which I don't think is part of the height. - to match notifs